plugin/src/main/java/com/denizenscript/denizen/objects/EntityTag.java

```diff
@@ -2873,13 +2875,52 @@ else if (object.getBukkitEntity() instanceof Hanging) {
         // A player's hand is raised when they are blocking with a shield, aiming a crossbow, looking through a spyglass, etc.
         // -->
         registerSpawnedOnlyTag(ElementTag.class, "is_hand_raised", (attribute, object) -> {
-            if (!(object.getBukkitEntity() instanceof HumanEntity)) {
+            if (!(object.getBukkitEntity() instanceof HumanEntity humanEntity)) {
                 attribute.echoError("Only player-type entities can have is_hand_raised!");
                 return null;
             }
-            return new ElementTag(((HumanEntity) object.getLivingEntity()).isHandRaised());
+            return new ElementTag(humanEntity.isHandRaised());
         });
 
+        if (NMSHandler.getVersion().isAtLeast(NMSVersion.v1_19)) {
+
+            // <--[tag]
+            // @attribute <EntityTag.last_attack>
+            // @returns MapTag
+            // @description
+            // Returns an interaction entity's last attack interaction, if any.
+            // The returned map contains:
+            // - 'player' (PlayerTag): the player who interacted

```

Comment:
This seems suspiciously like the time of case where it's a PlayerTag today but 1.21 or whatever will come out and it's an EntityTag now. Do we have reason to believe that isn't likely to happen?

Reply:
Only things that comes to mind is entities not really interacting with "random" stuff like that, as they have specific AI and whatnot, and Mojang largely having map making (I.e. mainly player related) stuff in mind with these features (hence the game time instead of a proper timestamp, for easier command block usage) - I guess we could convert it to an `EntityTag` that's always a player, although that could make script usage a bit more annoying; let me know if that should be done